pkgs/by-name/pr/proton-ge-custom/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, writeScript
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "proton-ge-custom";
+  version = "GE-Proton9-1";
+
+  src = fetchurl {
```

Comment:
Nit: I'd prefer

```suggestion
  src = fetchzip {
```

Reply:
I'd much prefer handling the unpacking of the tarball ourselves. If we use fetchzip, it'll be unpacked and we have to try and link more files to their correct paths before we are able to alter the manifest with paths.

Reply:
> we have to try and link more files to their correct paths before we are able to alter the manifest with paths.

I don't quite understand what you mean by that.

Reply:
We unpack to $out/opt and link necessary files to bin, then pointing to $out/opt as necessary. fetchzip unpacks everything to the out path, meaning we'd have to move all files locally *or* point to the base outpath.

Reply:
I still do not see how fetchzip + cp would be any different to fetchurl + manual untar into target.

With fetchzip, you could probably even just `ln -s`or `cp -rs`.

Reply:
But why? There are two steps to either options, so changing feels unnecessary.